scss/components/_newsletter-bar.scss

```diff
@@ -48,12 +48,12 @@
                 }
             }
         }
-        #button_block{
+        .button_block{
             width: 60%;
             @include md-screen{
                 width: 100%;
             }
-            #button-wrap{
+            .button_wrap{

```

Comment:
It's a bit confusing to have both `.button-wrap` and `.button_wrap` in separate places. It would be better to use one or the other.